templates/patroni.yml.j2

```diff
@@ -47,8 +47,11 @@ bootstrap:
       {% endfor %}
  
  #      recovery_conf:
-#        restore_command: cp ../wal_archive/%f %p
- 
+#        restore_command: cp ../wal_archive/%f %p #If you use wal-g, than use: wal-g wal-fetch %f %p
+      {% if postgresql_restore_command is defined and postgresql_restore_command | length > 0 %}
+        recovery_conf:
+          restore_command: {{ postgresql_restore_command }}
+      {% endif %}
```

Comment:
not correct, this does not apply to bootstrap.
You need to transfer this code to the "postgresql:" block, below "create_replica_methods - basebackup".